test/hotspot/jtreg/compiler/c2/Test8062950.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I'd argue this test serves no purpose now.

Reply:
Right, removed.